Modify kernel partition size for some built-in tools

chcyjoy opened the pull request:
Add openpower-pnor patch file to increase Bootloader Kernel partition size to 24MB for built-in RAID tool support

Signed-off-by: Joy Chu <joy_chu@wistron.com>

chcyjoy commented:
@klauskiwi Would you please help us again for this pull request? We plan to release a fw tomorrow.

--joy chu

HankChang commented:
@chcyjoy, please make this PR/Title more general as for extending the Mihawk PNOR kernel partition size in order to build-in the possible customized tools/binaries..., and then please also try to make it upon upstream first. 

Thanks!
-Hank

HankChang commented:
@klauskiwi  Just FYI - the same contents of pull-request on upstream:
https://github.com/open-power/op-build/pull/3283  (still reviewing)

Thanks, 
Hank

chcyjoy committed:
Modify: modify mihawk to support built-in tools

Add openpower-pnor patch file to increase Bootloader Kernel partition size to 24MB for built-in tools support

Signed-off-by: Joy Chu <joy_chu@wistron.com>
(cherry picked from commit dce2fd0998988eb5f3614ac8f6bc55730949bde6)
Signed-off-by: Joy Chu <joy_chu@wistron.com>

chcyjoy committed:
Modify: modify mihawk_defconfig for patch_dir

Add patch_dir define for the patch files

Signed-off-by: Joy Chu <joy_chu@wistron.com>
(cherry picked from commit 74277a3942b95f8f00fe682dee17b294fe3c82fc)
Signed-off-by: Joy Chu <joy_chu@wistron.com>

HankChang commented:
Upstream merged: https://github.com/open-power/op-build/pull/3283

klauskiwi commented:
@HankChang can you force-push this branch with the cherry-picked commit from upstream? "git cherry-pick -x -s" should do it.

chcyjoy commented:
@klauskiwi The cherry-pick commits from upstream are updated.
